app/src/main/res/layout/item_top_track.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.layout.item_top_track` appears to be unused
 Unused resources make applications larger and slow down builds.